commands/shooter/alignandaim.py

```diff
@@ -0,0 +1,150 @@
+from math import atan2, pi, sqrt
+import typing
+
+from commands2.command import Command
+from wpimath.controller import ProfiledPIDControllerRadians
+from wpimath.trajectory import TrapezoidProfileRadians
+from wpimath.geometry import Pose3d, Pose2d, Rotation2d, Translation3d, Translation2d
+from wpilib import DriverStation, SmartDashboard, Preferences
+
+
+from subsystems.shootersubsystem import ShooterSubsystem
+from subsystems.drivesubsystem import DriveSubsystem
+
+import constants
+from util.convenientmath import rotationFromTranslation
+
+
+class AlignAndAim(Command):
+    def __init__(
+        self,
+        shooterSubsystem: ShooterSubsystem,
+        driveSubsystem: DriveSubsystem,
+        forward: typing.Callable[[], float],
+        sideways: typing.Callable[[], float],
+    ) -> None:
+        Command.__init__(self)
+        self.setName(__class__.__name__)
+
+        self.shooter = shooterSubsystem
+        self.drive = driveSubsystem
+
+        self.forward = forward
+        self.sideways = sideways
+
+        self.targetPose = Pose3d()
+
+        self.thetaController = ProfiledPIDControllerRadians(
+            constants.kTrajectoryAnglePGain / 5,
```

Comment:
why /5 instead of just changing constant

Reply:
True!